### PR TITLE
Stats feature

### PR DESCRIPTION
**Pull request by isht3**
`evalai challenge CHALLENGE stats`

### CODE REVIEW

**Comment by RishabhJain2018** [comment]
@isht3 Please take a look at this error
screen shot 2018-07-16 at 6 12 15 pm
Also, please list down in the comments what all statistics you are showing.

**Comment by isht3** [comment]
@RishabhJain2018 Can you tell me how to reproduce this error? I'm not able to get it.

File: evalai/utils/challenges.py [hunk 1]
```diff
@@ -440,3 +440,94 @@ def display_leaderboard(challenge_id, phase_split_id):
...
+    challenge_phases = response["results"]
+
+    stats = {}
+    for phase in challenge_phases:
+        stats[phase["name"]] = []
+        count_stats_url = URLS.count_stats.value
+        url = "{}{}".format(get_host_url(), count_stats_url)
+        url = url.format(challenge_id, phase["id"])
+        try:
+            response = requests.get(url, headers=headers)
```
Comment (RishabhJain2018) [inline evalai/utils/challenges.py]: same as above!

File: evalai/utils/challenges.py [hunk 2]
```diff
@@ -440,3 +440,94 @@ def display_leaderboard(challenge_id, phase_split_id):
...
+def display_challenge_stats(challenge_id):
+    """
+    Function to display the stats of a challenge.
+    """
+    url = URLS.challenge_phase_list.value
+    url = "{}{}".format(get_host_url(), url)
+    url = url.format(challenge_id)
+    headers = get_request_header()
+    try:
+        response = requests.get(url, headers=headers)
```
Comment (RishabhJain2018) [inline evalai/utils/challenges.py]: Please add `verify=False`

File: evalai/utils/challenges.py [hunk 3]
```diff
@@ -440,3 +440,94 @@ def display_leaderboard(challenge_id, phase_split_id):
...
+            sys.exit(1)
+
+        response = response.json()
+        stats[phase["name"]].append(response)
+
+        last_submission_stats_url = URLS.last_submission_stats.value
+        url = "{}{}".format(get_host_url(), last_submission_stats_url)
+        url = url.format(challenge_id, phase["id"])
+        try:
+            response = requests.get(url, headers=headers)
```
Comment (RishabhJain2018) [inline evalai/utils/challenges.py]: same as above!

**Comment by RishabhJain2018** [comment]
@varunagrawal Can you please review it once!

**Comment by varunagrawal** [comment]
Didn't see the last comment. LGTM.

**Comment by RishabhJain2018** [comment]
@isht3 Can you please fix the bug here

screen shot 2018-07-31 at 6 35 53 pm

screen shot 2018-07-31 at 6 35 35 pm

In the first image in `Dev Phase` we have total 5 submissions but in the second image, we have 0. Moreover, please add a column for `Total Participant Teams`.

**Comment by RishabhJain2018** [comment]
@isht3 Can you please update this PR with the changes.

**Comment by RishabhJain2018** [comment]
@isht3 Can you please fix the conflicts and the bug in the PR.

**Comment by isht3** [comment]
@RishabhJain2018 I've fixed conflicts and added the stats feature. Can you check if it's okay? I'll update with the tests after that.

**Pull request closed without merging**